aten/src/ATen/native/ReflectionPadding1d.cpp

```diff
@@ -0,0 +1,227 @@
+#include <ATen/ATen.h>
```

Comment:
one last nit :)...I think it's slightly better if you call all these files ReflectionPad instead of ReflectionPadding; it's shorter and it matches the name of the function.

Reply:
Yes, will do!